CvGameCoreDLL_Expansion2/CvUnit.cpp

```diff
@@ -30283,770 +30295,771 @@ int CvUnit::AI_promotionValue(PromotionTypes ePromotion)
 			iValue += 1000;   // Enough to lock this one up
 		}
 	}
-
-#if defined(MOD_BALANCE_CORE_MILITARY_PROMOTION_ADVANCED)
+	*/
+#if defined(MOD_BALANCE_CORE_MILITARY_PROMOTION_ADVANCED)             // Don't know when if defined is needed
```

Comment:
From here downwards it would be better to just ignore the red bits as I rearranged it all to try and put similar promotion benefits together and put the currently unused (no pick-able promotions) benefits at the bottom.

Also every value calculation for each benefit which can be chosen has been changed.